### ISSUE TITLE
There are no nightly builds after 2018/11/04

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#1778]
Check https://sourceforge.net/projects/coolprop/files/CoolProp/nightly/

There are no nightly builds at all. Did we break this when releasing 6.2? Why did the builds stop on the 4th of November? 

This seems to be part of the problem described in #1762 .

**Comment by jowr** [comment]
Our hosting service runs a cron job that modifies permissions of suspicious files. Our executables look suspicious enough to trigger some actions and the modified file and folder permissions keep our release script from zipping up the docs.

**Comment by jowr** [comment]
I have update our own file permission scheme to be more restrictive. Let us hope that the restrictive permissions keep the hoster from assuming that our files are bad files. 

As soon as the OSX update for version 10.14.2 is done, I am going to fire up a full nightly build manually. Let us hope that all is well now.

**Comment by jowr** [comment]
After updating OSX I now fight the C++ standard library issue again. XCode 10 has completely removed support for libstdc++. See also https://github.com/jlfaucher/builder/commit/d144d3a695949f90c5e2acff4dfd94fdcf8dcdfa

@iobell: Should we finally retire the old settings and change the default value of "DARWIN_USE_LIBCPP" to "ON"?

**Comment by ibell** [comment]
@jowr - This is a non-backwards compatible change, right?  This would mean that if we make wheels on 10.14, they are no longer compatible on 10.8, for instance?  Or do I misunderstand?

**Comment by jowr** [comment]
I am not sure, maybe someone else can help here? However, I guess that the missing stdc++ libs in 10.14 should also break things, if there was an issue. 

The old wheels are all built using the std library and that does not exist on 10.14. Until now, I have not heard any complaints and I would thus guess that we do not have to satisfy any external dependencies, but who knows. No matter what, I would guess that wheels only for OSX >= 10.9 and above should be better than wheels for OSX <= 10.13 only.

By the way, I was super confused because I edited the CMake-based builds and nothing seemed to work until I noticed that there also is a non-CMake branch in the setup.py...

I think I implemented everything properly, but lets wait for the buildbots and we might also want to do some more research regarding the implications of 927970ba716edc501279d177c1919b65120143ac
